content/engineering/enablement/dev-experience/index.md

```diff
@@ -1,22 +1,28 @@
-# Developer Experience team
+# Dev Experience team
 
-The Developer Experience team (part of the [Enablement](../index.md) org) is a team focused on improving the developer experience of Sourcegraph.
+The Dev Experience team, or DevX for short, is a team focused on improving the developer experience of Sourcegraph as part of the [Enablement](../index.md) org.
 
 ## Members
 
 - [Taylor Sperry](../../../company/team/index.md#taylor-sperry) (Technical [Product Manager](../../../product/roles/index.md#product-manager))
-- [Patrick Dubroy](../../../company/team/index.md#patrick-dubroy) (acting [Engineering Manager](../../roles.md#engineering-manager))
+- Kristen Stretch ([Engineering Manager](../../roles.md#engineering-manager))
```

Comment:
@kstretch9 I don't think you have a section in our team page yet, i.e. https://handbook.sourcegraph.com/company/team !